labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -99,7 +98,8 @@ object ListManipulationExercise01 {
    * Implement it whatever way suites you best. Hint: it can be done in a neat way using recursion. 
    */
   def tails[T](l: List[T]): List[List[T]] = {
-    error("fix me")
+    if (l.isEmpty) List(List())
+    else l :: tails(l.tail)
   }
```

Comment:
could use pattern matching here! not really any slicker in this case but a fun tool to use